cli/command/container/cp.go

```diff
@@ -34,11 +39,74 @@ const (
 type cpConfig struct {
 	followLink bool
 	copyUIDGID bool
+	quiet      bool
 	sourcePath string
 	destPath   string
 	container  string
 }
 
+//cpWithProgress is a Wrapper to the content being copied from or to container. This wrapper
+//contains information about the file or folder being copied (size and bytes - ReadCloser) as well as
+//source and destination path. When the bytes are copied, the `Read` method from io.ReadCloser is overwritten
+//showing the progress during the copy.
+type cpWithProgress struct {
+	io.ReadCloser
+	toContainer   bool
+	total         int64
+	copySize      int64
+	containerName string
+	copyPath      string
+	quiet         bool
+	dockerCli     command.Cli
```

Comment:
Should just be an `io.Writer`

Reply:
Ok. Yeah, we don't have the total size in hands, so I'll update the loading to show only the amount transferred.

Reply:
Hello @cpuguy83 , I've made the changes in the code, can you review? I've removed the progress bar because we don't have the total size of file before the copy process and changed the output from `dockerCli` to `io.Writer`.